webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokensList/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PersonalAccessToken from './PersonalAccessToken';
+import { translate as __ } from '../../../../common/I18n';
+import { noop } from '../../../../common/helpers';
+
+const PersonalAccessTokensList = ({
+  title,
+  tokens,
+  revocable,
+  revokePersonalAccessToken,
+}) => (
+  <div className="panel panel-default">
```

Comment:
```suggestion
```

Reply:
I think `div` with `panel` class is necessary here